compressor/tests/test_offline.py

```diff
@@ -58,7 +61,12 @@ def setUp(self):
         # template to be skipped over.
         django_template_dir = os.path.join(settings.TEST_DIR, 'test_templates', self.templates_dir)
         jinja2_template_dir = os.path.join(settings.TEST_DIR, 'test_templates_jinja2', self.templates_dir)
-        settings.TEMPLATE_DIRS = (django_template_dir, jinja2_template_dir)
+
+        if django.VERSION >= (1, 8):
+            self.override_template_dirs = self.settings(TEMPLATE_DIRS=(django_template_dir, jinja2_template_dir))
```

Comment:
@diox Having had a quick look I'm not sure it's worth extracting the `template_dir` property. Yes, I can pull it out for the setting but it's used again inside the class in order to generate to path to directly render the template (and check it matches the `compress` output).